src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/PackageFeeds/MultiSourcePackageFeed.cs

```diff
@@ -111,7 +111,7 @@ public async Task<SearchResult<IPackageSearchMetadata>> SearchAsync(string searc
             }
 
             SearchResult<IPackageSearchMetadata> result;
-            using (var packageSourceTelemetry = new PackageSourceTelemetry(_sourceRepositories.Select(r => r.PackageSource), searchOperationId))
+            using (var packageSourceTelemetry = new PackageSourceTelemetry(_sourceRepositories, searchOperationId, "search"))
```

Comment:
"search" should be a constant? Maybe per Feed or in some common area in PackageSourceTelemetry 
 or perhaps the PackageManagement namespace?